frontend/src/views/Profile.vue

```diff
@@ -1,13 +1,16 @@
 <template lang="pug">
     v-row(justify="center")
+        HeaderToolbar
```

Comment:
This should not be here but jn `App.vue`

Reply:
changed